packages/webcert/src/store/utils/utilsMiddleware.ts

```diff
@@ -65,11 +69,29 @@ const handleGetDiagnosisTypeaheadSuccess: Middleware<Dispatch> = ({ dispatch }:
   dispatch(updateDiagnosisTypeahead(action.payload))
 }
 
+const handleGetConfig: Middleware<Dispatch> = ({ dispatch }: MiddlewareAPI) => (next) => (action: AnyAction): void => {
```

Comment:
Add test for this

Reply:
Same here, what logic is it that requires a test? Or phrased differently, what value does a test provide?

Reply:
The value that the test gives is verifying that the function works as intended and that future changes doesn't unintentionally change the functionality. You could write a test mocking a response from the endpoint. Then you could verify that the state has saved the correct values from the response that you mocked. You can use other middlewares and their tests for reference.